### PR TITLE
refine A2C example

### PR DESCRIPTION
**Pull request by zenghsh3**
- change CNN structure of A2C model
- add LinearDecayScheduler

### CODE REVIEW

File: parl/utils/scheduler.py [hunk 1]
```diff
@@ -14,12 +14,12 @@
 
 import six
 
-__all__ = ['PiecewiseScheduler']
+__all__ = ['PiecewiseScheduler', 'LinearDecayScheduler']
 
 
 class PiecewiseScheduler(object):
```
Comment (TomorrowIsAnOtherDay) [inline parl/utils/scheduler.py:20]: Set hyperparameters by a predefined step-based schedule.

File: parl/utils/scheduler.py [hunk 2]
```diff
@@ -38,18 +38,55 @@ def __init__(self, scheduler_list):
...
             if self.cur_step >= self.scheduler_list[self.cur_index + 1][0]:
                 self.cur_index += 1
                 self.cur_value = self.scheduler_list[self.cur_index][1]
 
-        self.cur_step += 1
-
         return self.cur_value
+
+
+class LinearDecayScheduler(object):
```
Comment (TomorrowIsAnOtherDay) [inline parl/utils/scheduler.py:64]: Set hyperparameters by a step-based schedule with linear decay values.

File: parl/utils/scheduler.py (hunk 1 above)
Comment (zenghsh3) [reply parl/utils/scheduler.py:20]: done

File: parl/utils/scheduler.py (hunk 2 above)
Comment (zenghsh3) [reply parl/utils/scheduler.py:64]: done

**Review by TomorrowIsAnOtherDay — APPROVED** [review]
Looks good to me.

**Pull request merged**